Makefile

```diff
@@ -342,11 +333,12 @@ tar: compile-with-docker
 
 clean-tar:
 	@rm -f $(TAR_LOC)/*.$(TAR_EXT)
-	@rm -f ${VERSION_FILE}
 
 # GITHUB_USER and GITHUB_TOKEN are needed be set to run github-release
-release: tar
-	TAR_FILENAME=$(TAR_FILENAME) TAR_FILE=$(TAR_FILE) \
-	OLD_VERSION=${OLD_VERSION} BUILD_VERSION=${BUILD_VERSION} \
+release-built-version: tar
```

Comment:
Was this meant to be release-built-version or does it refer to the built version of netplugin?

Reply:
this is a helper for release target, the release target sets BUILD_VERSION so that version is externally set instead of through scripts/getGitVersion.sh

Though I see I have BUILD_VERSION duplicated in the release target, should be able to remove one, will update